spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/SparkExpressionConverter.scala

```diff
@@ -199,6 +200,25 @@ trait SparkExpressionConverter {
         ConstantString("REGEXP_REPLACE") + blockStatement(convertStatement(expression.children.head, fields) + "," + s"r'${expression.children(1).toString}'" + "," + s"'${expression.children(2).toString}'")
       case _: FormatString | _: FormatNumber =>
         ConstantString("FORMAT") + blockStatement(convertStatements(fields, expression.children: _*))
+
+      case _ => null
+    })
+  }
+
+  def convertMathematicalExpressions(expression: Expression, fields: Seq[Attribute]): Option[BigQuerySQLStatement] = {
+    Option(expression match {
+      case _: Abs | _: Acos | _: Asin | _: Atan | _: Ceil |
+           _: Cos | _: Cosh | _: Exp | _: Floor | _: Greatest |
+           _: Least | _: Logarithm | _: Log | _: Log10 |
+           _: Pow | _:Round | _: Sin | _: Sinh |
```

Comment:
Does the Round expression here also take care when scale is given in the input? i.e. Does it take care of 
`case Round(child, scale)`

Reply:
Yeah it does Vinay. I have used `Round(child, scale)` function for many functions in the integration tests.
Like: `ROUND(LOG(word_count), 2)` `ROUND(ATAN(0.5),2)`